### DISCUSSION TITLE
An existing SAN certificate already contains the domain

### DISCUSSION THREAD

**Discussion by ashwinksharma** [rcl-ssl/docs#18] [Q&A] [answered]
Hi

I am trying to create a new standalone certificate and I am getting the following error:

ERROR: An existing SAN certificate already contains the domain *.humanshield.com.au. Please delete this certificate and try again

But the problem is that I can't delete the cert as someone who used to work for my organisation created the original sert - which has now expired.  

How do I move forward and create a new cert?  

Thanks

**Comment by rcladmin** [comment, marked as answer]
The expired certificate has been deleted. You can try to create a new one now.

**Comment by ashwinksharma** [comment]
Thanks.

Ashwin Sharma
CISO, CTO
Securemation Consulting
M: 0422120526
Suite 3, Level 5, 490 Upper Edward St, Springhill, QLD 4000

www.securemation.com<http://www.securemation.com/>
[small logo_edited]
***@***.******@***.******@***.***Partner[A picture containing text  Description automatically generated][Graphical user interface, application  Description automatically generated] ***@***.*** Certified ISO 27001 Lead Auditor

I respectfully acknowledge the Traditional Custodians of the land on which we live, work and do business. I would also like to pay my respects to the Elders past, present and emerging.
